### ISSUE TITLE
no such element: Unable to locate element: {"method":"class name","selector":"name"}

### ISSUE THREAD

**Issue by palewire** [joeyism/linkedin_scraper#5]
As of today, if I run this:

```python
company = Company("https://ca.linkedin.com/company/google")
```

This comes back:

```
NoSuchElementException: Message: no such element: Unable to locate element: {"method":"class name","selector":"name"}
  (Session info: chrome=64.0.3282.167)
  (Driver info: chromedriver=2.26.436382 (70eb799287ce4c2208441fc057053a5b07ceabac),platform=Linux 4.13.0-37-generic x86_64)
```

**Comment by joeyism** [comment]
Linkedin keeps changing protocol. You might need to login before scrapping. You can do that with 
```python
company = Company("https://ca.linkedin.com/company/google", scrape = False)
```
which will launch a webdriver. You can login, then run
```python
company.scrape()
```

**Comment by KUS5** [comment]
Even when logging in, I get this error message, do you know how to solve that?

**Comment by Pegovski** [comment]
Exactly same problem with the person to scrape, I guess css locators are changed or protocol

**Comment by Sl1nkman** [comment]
> 
> 
> Exactly same problem with the person to scrape, I guess css locators are changed or protocol

I believe it to be the CSS locators. 

For example, the About us description is now "about-us__description" and no longer "basic-info-description"

**Comment by ehsong** [comment]
Hi, I followed the instructions but I am getting a css_selector error:

`'NoneType' object has no attribute 'click'`

**Comment by joeyism** [comment]
This is fixed now as of [2.5.5](https://github.com/joeyism/linkedin_scraper/releases/tag/2.5.5)
